### PR TITLE
v1.0.1 templatetags issues

### PR DESCRIPTION
**Pull request by aj3sh**
- Fixed issues of django templatetags for v1.0.1

### CODE REVIEW

File: nepali/templatetags/nepalidatetime.py [hunk 1]
```diff
@@ -1,38 +1,158 @@
+import datetime
+from typing import Optional, Union
+
 from django import template
 from django.utils import timezone
 
+from nepali.datetime import nepalidate as _nepalidate
+from nepali.datetime import nepalidatetime as _nepalidatetime
 from nepali.datetime import nepalihumanize as humanize
```
Comment (sugat009) [inline nepali/templatetags/nepalidatetime.py:9]:
```suggestion
from nepali.datetime import (
    nepalidate as _nepalidate,
    nepalidatetime as _nepalidatetime,
    nepalihumanize as humanize
```

File: nepali/templatetags/nepalidatetime.py [hunk 2]
```diff
@@ -1,38 +1,158 @@
...
+    :returns: Nepali datetime format in 'en-US' locale
+    """
+    return nepalidate_en(datetime_obj, format=format)
 
 
 @register.filter(name="nepalidate_en")
-def nepalidate_en(datetime_obj, format="%B %d, %Y, %A"):
-    nepali_datetime_obj = to_nepalidatetime(datetime_obj)
-    if nepali_datetime_obj is None:
+def nepalidate_en(datetime_obj: _datetimes, format: str = _DEFAULT_DATE_FORMAT):
```
Comment (sugat009) [inline nepali/templatetags/nepalidatetime.py]: return type missing

File: nepali/templatetags/nepalidatetime.py [hunk 3]
```diff
@@ -1,38 +1,158 @@
...
+    """
+    try:
+        nepali_datetime_obj = to_nepalidatetime(datetime_obj)
+        return nepali_datetime_obj.strftime_en(format)
+    except InvalidNepaliDateTimeObjectException:
+        return None
+
+
+@register.filter(name="nepalidate_ne")
+def nepalidate_ne(datetime_obj: _datetimes, format: str = _DEFAULT_DATE_FORMAT):
```
Comment (sugat009) [inline nepali/templatetags/nepalidatetime.py]: return type missing

File: nepali/templatetags/nepalidatetime.py [hunk 4]
```diff
@@ -1,38 +1,158 @@
...
 @register.filter(name="nepalihumanize")
-def nepalihumanize(datetime_obj, threshold=None, format=None):
-    """templatetag to humanize nepalidatetime"""
-    nepali_datetime_obj = to_nepalidatetime(datetime_obj)
-    if nepali_datetime_obj is None:
+def nepalihumanize(
+    datetime_obj: _datetimes,
+    threshold: Optional[int] = None,
+    format: Optional[str] = None,
+):
```
Comment (sugat009) [inline nepali/templatetags/nepalidatetime.py]: return type missing

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: nepali/templatetags/nepalidatetime.py (hunk 2 above)
Comment (aj3sh) [reply nepali/templatetags/nepalidatetime.py]: Updated

**Review by sugat009 — APPROVED** [review]

**Pull request merged**